MaxText/configs/base.yml

```diff
@@ -101,6 +101,10 @@ jax_cache_dir: "~/jax_cache"
 # Hardware
 hardware: 'tpu' # Supported hardware types are 'tpu', 'gpu' and 'cpu'
 
+# Jax Initilization configuration
+process_id: -1
```

Comment:
please just directly pull this from env vars

Reply:
@rwittenI I opted not to set them as envvars because they are not global across all processes. Specifically, the values for `process_id` and `local_device_id` are specific to each process, whereas values like `COORDINATOR_PORT` and `COORDINATOR_ADDRESS` are more appropriately configured as envvars since they apply globally to all processes.

Please let me know if you have a different perspective.